build/configs/common_download.sh

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env bash
```

Comment:
Could you mention in the commit header that the common script is for imxrt and rtk only?

> build/configs : Add common download script _for imxrt1020-evk and rtl8721csm_

Reply:
This script is not exclusively for rtk and imxrt. It can be extended to other boards.

Reply:
Ohkay, thanks.